tf2_geometry_msgs/include/tf2_geometry_msgs/tf2_geometry_msgs.h

```diff
@@ -62,17 +62,17 @@ KDL::Frame gmTransformToKDL(const geometry_msgs::TransformStamped& t)
 // method to extract timestamp from object
 template <>
 inline
-  const builtin_interfaces::msg::Time& getTimestamp(const geometry_msgs::Vector3Stamped& t) {return t.header.stamp;}
+  const tf2::TimePoint& getTimestamp(const geometry_msgs::msg::Vector3Stamped& t) {return tf2_ros::fromMsg(t.header.stamp);}
```

Comment:
https://github.com/ros2/geometry2/pull/14 makes ROS 2 tf2 not return reference
https://github.com/ros/geometry2/pull/225 adds documentation to ROS 1 tf2 of the lifetime being tied to the argument lifetime